pkg/asset/agent/common.go

```diff
@@ -52,3 +58,48 @@ func IsSupportedPlatform(platform hiveext.PlatformType) bool {
 	}
 	return false
 }
+
+// OCexecute will execute an oc command.
+func OCexecute(executer executer.Executer, pullSecret, command string) (string, error) {
```

Comment:
@andfasano When I tried to use the existing `Execute`  in oc.go I would get import cycle issues when trying to refernece from installconfig.go  
```
package github.com/openshift/installer/cmd/openshift-install
	imports github.com/openshift/installer/cmd/openshift-install/agent
	imports github.com/openshift/installer/pkg/agent
	imports github.com/openshift/installer/pkg/asset/agent/image
	imports github.com/openshift/installer/pkg/asset/agent
	imports github.com/openshift/installer/pkg/asset/agent/image: import cycle not allowed
```
Moving this function to pkg/asset/agent/common.go seems to be the best compromise? Please advise.

Reply:
Sorry somehow I missed this one, let me check it better

Reply:
I think a reasonable approach could be the following:
* The `OCexecute` function could be moved to a new  `plg/asset/agent/oc.go` file. It depends only on system packages, and it will provide the pure functionality of the `oc` command, in its most generic form.
* The old `pkg/asset/agent/image/oc.go` must be renamed to something else (I'd suggest `releaseextract.go`) since it's currently offering just two funcs to extract stuff from a release: a generic `ExtractFile` and a more specialized `GetBaseIso`